### ISSUE TITLE
Edit Recording Schedules Menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#119]
```
Menu for editing or adding a recording (either from the recording schedules or 
adding a new one from the tv-guide) gives me some strange strings. Look at the 
attached images.

--
Ubuntu 10.04.1 LTS

--
MythTV Version   : 24158
MythTV Branch    : branches/release-0-23-fixes
Network Protocol : 56
Library API      : 0.23.20100314-1
QT Version       : 4.6.2

--
XBMC 10.0-beta1 r333324

Screenshots provided.
```

Original issue reported on code.google.com by `misfi...@gmail.com` on 10 Sep 2010 at 11:39

Attachments:
- [myscreen1.png](https://storage.googleapis.com/google-code-attachments/mythbox/issue-119/comment-0/myscreen1.png)
- [myscreen2.png](https://storage.googleapis.com/google-code-attachments/mythbox/issue-119/comment-0/myscreen2.png)

**Comment by GoogleCodeExporter** [comment]
```
Oh. Forgot to tell what version of mythbox im using. It's RC2 from the svn. :D
```

Original comment by `misfi...@gmail.com` on 13 Sep 2010 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
haven't has a chance to do any testing on dharma. will get to it eventually..
```

Original comment by `semir.pa...@gmail.com` on 28 Sep 2010 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.0
```

Original comment by `semir.pa...@gmail.com` on 26 Nov 2010 at 3:27
- Changed state: **Fixed**
